Marlin/src/feature/host_actions.cpp

```diff
@@ -182,7 +182,7 @@ void host_action(PGM_P const pstr, const bool eol) {
         break;
       case PROMPT_PAUSE_RESUME:
         msg = PSTR("LCD_PAUSE_RESUME");
-        #if ENABLED(ADVANCED_PAUSE_FEATURE)
+        #if BOTH(ADVANCED_PAUSE_FEATURE,SDSUPPORT)
```

Comment:
A space is missing. 😛

Reply:
That fixed my problem.
Thanks a ton. 👍

Reply:
Ill get right on that, its the most important thing in the world to have just the correct amount of spaces. (apparently)